src/scheduler/pbsfs.c

```diff
@@ -60,12 +60,6 @@
 #include "sched_cmds.h"
 #include "log.h"
 
-
-
-/* to make references happy */
-int pbs_rm_port;
-int got_sigpipe;
```

Comment:
I moved them to globals.c instead. This will allow somebody to link against libpbs_sched.a alone and utilize scheduler functionality without getting undefined references. If it's just to make references happy then why not put it in libpbs_sched.a itself? Programs like pbs_sched can still have their own pbs_rm_port & got_sigpipe and pre-set & actually use them just like before. What do you think?